datasets/br/suspended_companies/crawler.py

```diff
@@ -0,0 +1,111 @@
+from zavod import Context
+from zavod import helpers as h
+from zipfile import ZipFile
+import csv
+import io
+import re
+from typing import List
+
+SITE_URL = "https://portaldatransparencia.gov.br/download-de-dados/ceis"
```

Comment:
I would use `context.data_url` instead (which is `data.url` in the YAML metadata!) because it means that if the metadata is outdated, the code breaks...